modules/rsyslog/manifests/init.pp

```diff
@@ -36,6 +36,14 @@
             require => [ Package['rsyslog'], File[$server_logdir] ],
         }
 
+        file { "/usr/local/bin/tail-hilary":
+            content => template('rsyslog/tail-hilary.erb'),
+            owner   => 'root',
+            group   => 'root',
+            mode    => 0754,
+            require => [ Package['rsyslog'], File[$server_logdir] ],
```

Comment:
`tail-hilary` and `filter-bunyan` no longer require these things anymore if they're in /usr/local/bin, so we can probably remove these dependencies.